astropy/time/formats.py

```diff
@@ -710,7 +710,7 @@ class TimeUnix(TimeFromEpoch):
 
 class TimeUnixTai(TimeUnix):
     """
-    Seconds from 1970-01-01 00:00:08 TAI (see notes), including leap seconds.
+    Counts the number of SI seconds elapsed since 1970-01-01 00:00:08 TAI.
```

Comment:
Please revert this unrelated change, the original is consistent in style with the other format class descriptions.

Reply:
The point is that the formulation `including leap seconds` is wrong.

Monotonic timescales do not have leap seconds, that's the point of them.

Leap seconds are a property of UTC.

Reply:
I reverted the changes here and will add a second PR for those